app/src/DbConnector.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App;
+
+/**
+ * Class DbConnector
+ * @package App
+ * class to connect to database
+ * dependencies: needed for all database queries.
+ */
+class DbConnector
+{
+    public $db;
+
+    /**
+     * DbConnector constructor.
+     * * automatically instantiates the database connector object using PDO which can then be called with the $db variable.
```

Comment:
Remove one of the stars on this line and use multiline if this sentence is too long.
When referring to $db, please use the word property instead of variable